sisl/io/siesta/ani.py

```diff
@@ -8,11 +8,15 @@
 from ..sile import add_sile
 from sisl._internal import set_module
 
+
 __all__ = ["aniSileSiesta"]
 
+
 @set_module("sisl.io.siesta")
 class aniSileSiesta(xyzSile):
 
-    pass
+    def read_geometry(*args, all=True, **kwargs):
+        return super().read_geometry(*args, all=all, **kwargs)
```

Comment:
I'm not sure that `all` should default to `True` here. `read_geometry` sounds like it's going to read just one geometry :sweat_smile:

Reply:
But I don't want a function which reads 1, and one which _can_ read more. That seems like a waste. The same is the case for `outSileSiesta` for instance.

Since the ANI file is intrinsically having _many_ coordinates, and users would probably expect to read everything, then I think having `all=True` being the expected behaviour.